tests/texture.py

```diff
@@ -425,6 +425,8 @@ def texture_2d_array_mipmap(cfg: ngl.SceneCfg):
 void main()
 {
     float x = floor(var_uvcoord.x * steps) / steps;
+    float t = step(0.5, var_uvcoord.y);
+    x = mix(x, 1.0 - x, t);
```

Comment:
can you explain what you are doing here? I tried to graph what's going on here, and it looks like a slope rotating non linearly around (½,½), but that's not exactly helping.

Reply:
I just want the original color x for the upper part of the texture [0 -> 0.5] and its inverse for the lower part [0.5, 1].

Reply:
ah ok yeah so `x = var_uvcoord.y < 0.5 ? x : 1 - x` 😅

Reply:
yup.

Reply:
Adjusted to this more readable form.